package-lock.json

```diff
@@ -387,6 +387,12 @@
         "sprintf-js": "~1.0.2"
       }
     },
+    "argv": {
```

Comment:
Since no packages are being added or removed, should we be diffing this file?

Reply:
In this case the `codecov` package is being added, but I agree we that package.json is still causing trouble occasionally. What would you recommend? `git rm`, or just remove it from the commit?

Reply:
Ahh .. I missed that totally. Thanks for clarifying. I think its good / or ok since we are changing a package.